Formula/osinfo-db.rb

```diff
@@ -0,0 +1,17 @@
+class OsinfoDb < Formula
+  desc "Osinfo database of information about operating systems for virtualization provisioning tools"
```

Comment:
Audit failure:

```
        Error: 1 problem in 1 formula
osinfo-db:
  * C: 2: col 3: Description is too long. "name: desc" should be less than 80 characters. Length is calculated as osinfo-db + desc. (currently 103)
```

Can this be shortened?

Reply:
I've removed the formula per next request.